less/index.less

```diff
@@ -4,6 +4,19 @@
    License: none (public domain)
 */
 
+@import "colors.less";
+@import "buttons.less";
```

Comment:
Good job with importing an external LESS module.
As you already might now now, you could want to break your project in even more LESS modules.